pages/apis/rest_api/pipelines.md.erb

```diff
@@ -332,6 +332,130 @@ Optional [request body properties](/docs/api#request-body-properties):
 </tbody>
 </table>
 
+Provider settings properties
+
+Bitbucket Cloud, Bitbucket Server, GitHub, and GitHub Enterprise all have optional `provider_settings`. Other source providers such as Gitlab have no optional settings; however they will return the read-only `repository` field in the `provider_settings` object of a pipeline REST API response. 
```

Comment:
The readonly `repository` field section could be a bit confusing, because the rest is talking about the params you pass in, and this is talking about the response? I was wondering what the response looks like too? It is `{"provider_settings": {"repository": "git@myserver.com"}}` type thing?

Reply:
Ah right. I didn't realise that was part of the response, that makes much more sense! Do you think we need to include a separate example response for this section? It is listed up in the create provider response:

![image](https://user-images.githubusercontent.com/2074469/67291911-2b6b2200-f4da-11e9-969b-f2252b8f0406.png)

Reply:
Oh, clearly I did realise at the time I wrote it given that it says that it's part of the response 🤦🏻‍♀️I'll just remove that bit I think.